# Metabolomics analysis report
SBL.20004 

18.05.2022

----
Group C
- Christelle Brumann
- Gloria Lampo
----


## Introduction

Some words on the backgorund of your projects.
Which plants did you select and why ?

The aim of this report is to study the metabolome of several Salvia strains cultivated in the botanical garden of the University of Fribourg. The ambition is to reconstruct a phylogenetic tree based on the metabolome’s analysis and compare it to databases and similar studies. Data collected will contribute to the molecular cartography of the botanical garden. This project is led by Dr. Emmanuel Defossez and Dr. Pierre-Marie Allard [^1]. 

The study of related plants’s metabolome approaches phylogeny from an innovative perspective. It opens up investigation tracks for understanding the diversity of species we encounter in nature. Mass spectrometry(MS) -based plant metabolomic should be as efficiant as sequencing. It's what we want to experiment with our porject. Firstly it’s important to think about which type of metabolite can help to solve our biological question.

Plants are sessile and cannot escape to their environment. They must adjust their metabolism to both abiotics and biotic stress. Metabolites produced may therefore be likely different in the same plant living in an inhospitable environment than in a hospitable environment. This hypothesis raises the impact of the choice of metabolites analysed. In our study we bypass this potential source of error. Salvia sampling was made in the same area of the botanical garden. It means that we assume that plants have been subjected to identical stress. Of course, this does not simplify the selection of metabolites potentially suitable to detect phylogenetic links between species. 

Interesting plant metabolites have already been measured by MS and listed [2]. Based on the observations made the spatio-temporal distribution of metabolites in the plants could be pointed out. This underlines the importance of sampling points on the plant. For our experiment, we systematically sampled  leaves. It would also have been interesting to take the roots of each plant but for a concern of preservation of our study specimen we did not collect the roots. 

At this point the harvest strategy is defined and the wet lab part starts. The solvent used for the extraction is also a delicate concern because it can lead to a metabolit selection before MS-analysis. The extraction solution used in this study is MeOH/H2O/formic acid (80:20:0.1). Hydrophilic compounds were extracted with polar organic solvent such as methanol because of its affinity for water molecules that leads to a hydrophilic molecules rearrangement. A low concentration of acid such as formic acid helps the ionisation of molecules and it implies that analytes become more basic than solvent [3].  As explained, the extraction step modifies our sample and proceeds to a metabolites sorting during extraction steps. So in this study the harvest and extraction steps were performed at the same time and  in the same way for all samples to reduce variability between samples. 

After extraction and before MS-analysis an additional step of separation is executed: a high performance liquid chromatography (HPLC). Briefly explained, metabolites pass through a solid column driven by a mobile phase. This leads to an elution at different speeds based on their affinity properties with the solid column and the mobile phase. At the end of the column molecules come through with a specific retention time according to their electronic properties [4]. 
The final step of the wet lab part strategy is the MS. This technique has opened up the perspectives of understanding the global molecular cell dynamics and is commonly used in proteomic and metabolomic research. These studies are closely related since many metabolite’s transformations are induced by enzymes which are proteins.

The machine is composed of three elements: an ion source, mass analyser and detector. The first element of the MS performs an electrospray ionisation. It means that the analytes are sprayed through a needle with high electrical potential to be ionised before entering in the analyser chamber. Time of flight analyzer (TOF) was used in this study. It is built by a combination of a liner trap. It determines the m/z ratio [5]. The analyzer used …. 
Sample

The dry lab part … [6] [7]

Among the genus Salvia from the lamiaceae family, we collected 14 plants of 11 different species. They represent 3 clades subgenus (Glutinaria, Salvia and Sclarea) of the 11 commonly accepted. Heterosphae is not clearly defined as a clade but its heritage differs as a result of geographical events. A genomic study using next-generation sequencing on which we base our comparison, defined Heterosphae as a clade [8]. To simplify and do easier comparisons we will use the same nomenclature. 


## Material & Methods

### Sample collection

- Where we the plants collected ? 
- Which species were collected ? 
- Link to the iNaturalist entries of your species.

You can link a csv that you upload on github.
Go to the export link of the DBGI project on inaturalist https://www.inaturalist.org/observations/export?projects=digital-botanical-gardens-initiative
Apply eventual filters (fo rexample by username)
Save the csv and upload to github.


Example see [table X](https://github.com/commons-teaching/SBL.20004.2022/blob/main/data/observations-238383.csv) 

### Sample preparation

- Describe your sample preparation

### LCMS Analysis

- Describe the LC conditions
- Describe the MS conditions

### Data treatment

- Describe the software and parameters used.

## Results


### MS1

How many features could you clean in your final peak list ?
A link to the final feature list (uploaded to github).

### Molecular Network

Screenshots of your molecular network and of some clusters of interest.
Link to the GNPS job.
Link to the GNPS identification table.

## Conclusion

Some conclusion that you could get out of this preliminary study.

# References

[^1] Kozlowski Gregor, Defossez Emmanuel, & Allard Pierre-Marie. (2021). Molecular cartography of the Botanical Garden of the University of Fribourg (1.1). Zenodo. https://doi.org/10.5281/zenodo.5726749

[^2] Shouchuang Wang, Saleh Alseekh, Alisdair R. Fernie, Jie Luo. (2019). The Structure and Function of Major Plant Metabolite Modifications, Molecular Plant,  vol 12, issue 7, P899-919, https://doi.org/10.1016/j.molp.2019.06.001

[^3] Jorge Tiago F., Mata Ana T. & António Carla. (2016). Mass spectrometry as a quantitative tool in plant metabolomics, Phil. Trans. R. Soc. A.3742015037020150370
http://doi.org/10.1098/rsta.2015.0370

[^4] Malviya, Rishabha & Bansal, Vvipin & Pal, Om & Sharma, Pramod. (2010). High performance liquid chromatography: A short review. Journal of Global Pharma Technology. 2. 22-26. 

[^5] Walther TC, Mann M. Mass spectrometry-based proteomics in cell biology. J Cell Biol. 2010 Aug 23;190(4):491-500. 
https://doi.org/10.1083/jcb.201004052

[^6]

[^7]

[^8] Rose Jeffrey P., Kriebel Ricardo, Kahan Larissa, DiNicola Alexa, GonzÃ¡lez-Gallegos JesÃºs G., Celep Ferhat, Lemmon Emily M., Lemmon Alan R., Sytsma Kenneth J. & Drew Bryan T. (2021). Sage Insights Into the Phylogeny of Salvia: Dealing With Sources of Discordance Within and Across Genomes, Frontiers in Plant Science, Vol..12	
https://doi.org/10.3389/fpls.2021.767478   

[^9]

[^10] Etat de Fribourg, Energy, agriculture et environnement: Degrés-jours et température moyenne, relevés hebdomadaires, website visited on 27 April 2022 
https://www.fr.ch/energie-agriculture-et-environnement/energie/degres-jours-et-temperature-moyenne-releves-hebdomadaires

[^11] INaturalist, website visited on 13 April 2022
DBGI project https://www.inaturalist.org/projects/digital-botanical-gardens-initiative

[^12] Metabolomic data https://drive.switch.ch/index.php/s/U6Vezrq5mDVzSld

[^13] Proteowizard https://proteowizard.sourceforge.io/

[^14] MzMine 2.53 https://github.com/mzmine/mzmine2/releases

[^15] Filezilla https://filezilla-project.org/download.php?type=client

[^16] Cytoscape https://cytoscape.org/

Note that you can make a footnot like this [^1]

[^1]: Ref X
